toolchain/sem_ir/stringify_type.cpp

```diff
@@ -202,9 +207,11 @@ auto StringifyTypeExpr(const SemIR::File& outer_sem_ir, InstId outer_inst_id)
         }
         break;
       }
-      case CARBON_KIND(FacetTypeAccess inst): {
-        // Print `T as type` as simply `T`.
-        push_inst_id(inst.facet_id);
+      case CARBON_KIND(FacetValue inst): {
+        // No need to output the witness.
+        push_inst_id(inst.type_inst_id);
+        push_string(" as ");
+        push_inst_id(sem_ir.types().GetInstId(inst.type_id));
```

Comment:
Should this be:
```suggestion
        push_inst_id(sem_ir.types().GetInstId(inst.type_id));
        push_string(" as ");
        push_inst_id(inst.type_inst_id);
```
... because we process the pushed parts in reverse order?

Reply:
I applied this change, but it seems this isn't covered by any of our tests. When would this type be stringified?

Reply:
Added the test you suggested, which seemed to have worked!